policy/release/cve_test.rego

```diff
@@ -30,7 +30,28 @@ test_success if {
 	lib.assert_empty(cve.deny) with input.attestations as attestations
 }
 
-test_success_with_rule_data if {
+test_success if {
+	slsav1_task_with_result := tkn_test.slsav1_task_result_ref(
+		"clair-scan",
+		[{
+			"name": cve._deprecated_result_name,
```

Comment:
Given the test name, this should probably be `cve._result_name` and the test below should use `cve._deprecated_result_name`. (Or just swap the test names)

Reply:
Fixed!